routes/markerSites.js

```diff
@@ -125,7 +130,13 @@ module.exports = function(connection) {
 
     /* Update marker with given id */
     router.post('/marker/:id', isAuthenticated, function (req, res, next) {
+
         req.body = JSON.parse(Object.keys(req.body)[0]);
+
+        //If authenticated user is not owner of the marker
+        //and is not admin, return 401 (Unauthorized)
+        if(req.session.user.uid != req.body.uid && !req.session.user.admin) return res.send(401);
```

Comment:
Is there are a reason to use `!=` instead of `!==`?

Reply:
@bmamlin Replaced both "!=" with "!==". :)